code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -232,3 +232,27 @@
 	desc = "These boots were made for dancing."
 	icon_state = "bsing"
 	put_on_delay = 50
+
+/obj/item/clothing/shoes/cowboyboots

```

Comment:
we already have brown cowboy boots; now need for duplicate.

Also, again, all color varients should be a subtype.